frontend/src/api/service/LadeMessdatenService.ts

```diff
@@ -0,0 +1,48 @@
+import FetchService from "@/api/service/FetchService";
+import LadeBelastungsplanDTO from "@/types/zaehlung/zaehldaten/LadeBelastungsplanDTO";
+import OptionsDTO from "@/types/zaehlung/OptionsDTO";
+import LadeProcessedZaehldatenDTO from "@/types/zaehlung/zaehldaten/LadeProcessedZaehldatenDTO";
+import LadeZaehldatenZeitreiheDTO from "@/types/zaehlung/zaehldaten/LadeZaehldatenZeitreiheDTO";
+
+export default class LadeMessdatenService {
+    private static readonly ENDPOINT_LADE_MESSDATEN_PROCESSED: string =
+        "api/dave-backend-service/lade-messwerte";
+
+    private static readonly ENDPOINT_LADE_BELSTUNGSPLAN: string =
+        "api/dave-backend-service/lade-belastungsplan";
+
+    private static readonly ENDPOINT_LADE_ZEITREIHE: string =
+        "api/dave-backend-service/lade-zeitreihe";
+
+    public static ladeMessdatenProcessed(
+        messstelleId: string
+    ): Promise<LadeProcessedZaehldatenDTO> {
+        return FetchService.getData(
+            `${this.ENDPOINT_LADE_MESSDATEN_PROCESSED}?messstelle_id=${messstelleId}`,
+            "Beim Laden der aufbreiteteten Messdaten ist ein Fehler aufgetreten."
+        );
+    }
+
+    public static ladeBelastungsplan(
```

Comment:
bitte in zukunft nur das machen, was auch im ticket verlangt ist. Ich weiß wir brauchen das vermutlich später, aber das erwarte ich eigentlich in nem anderen MR. Können das jetzt drinnen lassen, aber in zukunft im passenen MR machen, dann ist das Review auch leichter

Reply:
Ja das war dem Copy-Paste geschuldet. Sorry